### ISSUE TITLE
Add. 14,667, foll. 50, 51 | https://bl.vuexistapps.us/manuscript/472 (Part11)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1125]
### Problems with Manuscript Overview

- In the MS overview, "A.D. 1057 / A. Gr. 1368" is mentioned as the date. However, according to Wright's description, the manuscript contains a note that includes "a series" of dates, and "A.D. 1057 / A. Gr. 1368" is the latest. So please check if the date in the MS overview should remain the same. 

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### General Notes

The Syriac writing in the Additions section is misordered.

**Comment by mikey-tang16** [comment]
Question @wlpotter 
Since Wright doesn't actually give any other date, I assume we can keep this date in the overview?

**Comment by wlpotter** [comment]
Yes, we can leave the date in the overview. The note records multiple dates, though Wright only tells us what the latest one is. I think we can assume, though, that that is probably when the note was written (or near enough)
